### ISSUE TITLE
How is the validation set divided?

### ISSUE THREAD

**Issue by Hello-2073** [Ekko-zn/AIGCDetectBenchmark#32]
I noticed `validate()` and `early_stopping()` methods in `train.py` file, which requires `dataset/val/` to work. However, how to divide the validation is not mentioned in the paper and code.  Could you help me find the answer here?
